airbyte-cdk/java/airbyte-cdk/typing-deduping/src/test/kotlin/io/airbyte/integrations/base/destination/operation/AbstractStreamOperationTest.kt

```diff
@@ -85,7 +85,7 @@ class AbstractStreamOperationTest {
             val initialState =
                 mockk<DestinationInitialStatus<MinimumDestinationState.Impl>> {
                     every { streamConfig } returns this@Truncate.streamConfig
-                    every { initialRawTableStatus } returns mockk<InitialRawTableStatus>()
+                    every { initialRawTableStatus.rawTableExists } returns false
```

Comment:
these diffs are just to handle the new behavior. They don't affect the test in any relevant way.